networkx/algorithms/connectivity/connectivity.py

```diff
@@ -771,7 +772,7 @@ def edge_connectivity(G, s=None, t=None, flow_func=None):
             return 0
 
         # initial value for \lambda is minimum degree
-        L = min(G.degree().values())
+        L = min(dict(G.degree()).values())
```

Comment:
Would this use less memory (and maybe faster)?

    L = min( d for n,d in G.degree() )